docker/graphdb/Dockerfile

```diff
@@ -4,4 +4,4 @@ RUN mkdir -p /root/graphdb-import/data
 WORKDIR /root/graphdb-import/data
 
 COPY ./initdb /initdb
-RUN cd /initdb  && ./init_graphdb.sh 
\ No newline at end of file
+RUN cd /initdb  && sh ./init_graphdb.sh 
```

Comment:
this still needed after the line-ending-fixing?
removing it brings interpreter selection back under responsibility of script (and is set to bash)